lambdas/backend-client/src/schemas/client.ts

```diff
@@ -5,7 +5,6 @@ import { ClientConfiguration, ClientFeatures } from '../types/generated';
 const $ClientFeatures = schemaFor<ClientFeatures>()(
   z.object({
     proofing: z.boolean(),
-    // TODO: CCM-11148 Make routing required
```

Comment:
Just based on what this TODO is saying - should this still be flagged as optional?